### PR TITLE
fix: set lowercase column names

### PR DESCRIPTION
**Pull request by FelixMoelder**
* fix: column names need to be lowercase for being in line with the latest `dna-seq-varlociraptor` release.
* fix: do not remove any splicing related candidate variant

### CODE REVIEW

**Comment by FelixMoelder** [comment]
Before merging we should wait for a new `dna-seq-varlociraptor` release including https://github.com/snakemake-workflows/dna-seq-varlociraptor/pull/247

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**
